src/qibolab/instruments/emulator/pulse_simulator.py

```diff
@@ -35,6 +36,11 @@
 
 GHZ = 1e9
 
+# noise_model
+SNR = 30  # dB
+NOISE_AMP = np.power(10, -SNR / 20)
+AWGN = lambda t: np.random.normal(loc=0, scale=NOISE_AMP, size=len(t)) * 3e4
+
```

Comment:
This shouldn't be hardcoded as the SNR is setup dependent. Maybe this can be set up in the platform script?